V/Valhalla/bundled/patches/pkg-config-mingw.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 4632fdbcf..c76b29884 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -291,7 +291,7 @@ if(PKG_CONFIG_FOUND)
+   if(NOT \${_res} EQUAL 0)
+     message(FATAL_ERROR \"Configuring libvalhalla.pc failed: \${_err}\")
+   endif()")
+-  install(FILES ${VALHALLA_BINARY_DIR}/libvalhalla.pc
++  install(FILES ${VALHALLA_BINARY_DIR}/libvalhalla.pc.exe
```

Comment:
:raised_eyebrow:

Reply:
The file that is created is `pc.exe`...and its windows so it seemed plausible to me that we should accommodate it. Or is the error actually that a `.pc.exe` file is generated? In which case is something going wrong here? https://github.com/valhalla/valhalla/blob/f7c31e5c232dd20156235ef1ba2184dc5a27309a/src/CMakeLists.txt#L290

Reply:
I don't see why a pkg config file should have any extensions but `.pc`, on ***any*** operating system.

Reply:
Ok, so I think I'm going to accept that the Valhalla pkg-config setup for windows is broken and beyond my ability to debug. I've raised the issue upstream: https://github.com/valhalla/valhalla/issues/3931